File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleImageAndTextElement.java
package com.gigigo.orchextra.core.domain.entities.article;

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement;
import java.io.Serializable;
import java.util.List;

public class ArticleImageAndTextElement extends ArticleElement implements Serializable {

  private String text;
  private String imageUrl;
  private List<Float> ratios;

  public String getText() {
    return text;
  }

  public void setText(String text) {
    this.text = text;
  }

  public String getImageUrl() {
    return imageUrl;
  }

  public void setImageUrl(String imageUrl) {
    this.imageUrl = imageUrl;
  }

  public List<Float> getRatios() {
    return ratios;
  }

  public void setRatios(List<Float> ratios) {
    this.ratios = ratios;
  }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleImageAndTextElement.kt
package com.gigigo.orchextra.core.domain.entities.article

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement
import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElementRender

class ArticleImageAndTextElement : ArticleElement<ArticleImageAndTextElementRender>()

class ArticleImageAndTextElementRender : ArticleElementRender() {

  var text: String? = null
  var imageUrl: String? = null
  var ratios: List<Float>? = null
}
